vendor/bat-native-ads/src/bat/ads/internal/catalog/catalog_unittest.cc

```diff
@@ -28,6 +30,18 @@ constexpr char kCatalogWithSingleCampaign[] =
 constexpr char kCatalogWithMultipleCampaigns[] =
     "catalog_with_multiple_campaigns.json";
 
+std::string ReadFileAndParseTagsToString(const std::string& name) {
+  const absl::optional<std::string>& content_optional =
+      ReadFileFromTestPathToString(name);
+  if (!content_optional.has_value()) {
+    return "";
+  }
+
+  std::string content = content_optional.value();
```

Comment:
we are making copy of string here. to avoid this we could get make `content` as pointer to `content_optional.value();`

Reply:
Fixed